src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -98,24 +98,42 @@ public static class TextCatalog
 
         /// <summary>
         /// Extracts word embeddings.
+        /// Output three times more values than dimension of the model specified in <paramref name="modelKind"/>
+        /// First set of values represent minumum encountered values (for each dimension), second set represent average (for each dimension)
+        /// and third one represent maximum encountered values (for each dimension).
         /// </summary>
```

Comment:
Instead of the summary, you can also reference the section in the doc.xml, through <include file>

 /// <include file='doc.xml' path='doc/members/member[@name="WordEmbeddings"]/*' />